.github/workflows/test.yaml

```diff
@@ -13,10 +13,6 @@ env:
   KEYCLOAK_CLIENT_ID: ${{ secrets.KEYCLOAK_CLIENT_ID }}
   KEYCLOAK_CLIENT_SECRET: ${{ secrets.KEYCLOAK_CLIENT_SECRET }}
   NEXTAUTH_SECRET: ${{ secrets.NEXTAUTH_SECRET }}
-  CHES_CLIENT_ID: ${{ secrets.CHES_CLIENT_ID }}
```

Comment:
thanks for having my back Sepehr! I forgot to remove these in my PR _even after you reminded me about it_! 🤦‍♀️

Reply:
Anytime 🥷